client/src/pages/Editor.tsx

```diff
@@ -71,11 +76,17 @@ export class Editor extends Component<IProps, IState>
                 timeout: 0,
                 show: false,
             },
+            splittingAmmount: 50,
```

Comment:
Could we use `localStorage` to save/get this amount? That way when the user comes back, it's the same layout when they left.

Reply:
splittingAmmount => splittingAmount

Reply:
Done!